src/components/SendTokens/EthereumTokens.js

```diff
@@ -523,19 +542,22 @@ class SendEthereumTokens extends React.Component<Props, State> {
       return (
         <TouchableOpacity onPress={() => this.setState({ showTransactionSpeedModal: true })}>
           <SendTokenDetailsValue>
-            <BaseText secondary>Fee: {this.getTransactionFeeString()}</BaseText>
+            <BaseText center secondary>Fee: {this.getTransactionFeeString()}</BaseText>
           </SendTokenDetailsValue>
+          {!!feeError &&
+            <BaseText center secondary>Not enough ETH for the transaction fee</BaseText>
```

Comment:
Could you make the message a bit friendlier please?

```suggestion
            <BaseText center secondary>Sorry, you do not have enough ETH in your wallet to make this transaction. Please top up your wallet and try again.</BaseText>
```